core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/connection/DatabaseConnectionService.java

```diff
@@ -17,8 +17,14 @@
     public Connection getConnection(T databaseConnection) {
         try {
             Class.forName(getDriver(databaseConnection));
-            Connection connection = DriverManager.getConnection(databaseConnection.getConnectionURL(), databaseConnection.getUserName(), databaseConnection.getUserPassword());
-            connection.setAutoCommit(false);
+            Connection connection;
+            //Ensure Bigquery connection url can be used
+            if (databaseConnection.getType().equalsIgnoreCase("bigquery")) {
+                connection = DriverManager.getConnection(databaseConnection.getConnectionURL());
+            } else {
+                connection = DriverManager.getConnection(databaseConnection.getConnectionURL(), databaseConnection.getUserName(), databaseConnection.getUserPassword());
+                connection.setAutoCommit(false);
+            }
```

Comment:
Method 'public Connection getConnection(T databaseConnection)' should be overriden in 'core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/bigquery/BigqueryDatabaseConnectionService.java'

Reply:
Resolved in next commit